src/script.ts

```diff
@@ -168,6 +179,10 @@ window.onload = function() {
     }
     setTimeout(location.reload, reloadTimer);
 
+    // Update the clock every second
+    setInterval(updateClock, 1000);
```

Comment:
no point in burning CPU updating every second if we're only displaying in minute precision

Reply:
Yeah, but if not every second what if we start at the 30th second of a minute. Probably a `setTimeout` so that `setInterval` can run at the first second of a minute?

Reply:
Updating the clock every second barely burns CPU, considering we make an HTTP request every second anyways this seems like it would be a premature optimization. Second frequency is fine here.